libs/components/forms/src/lib/modules/input-box/input-box-hint-text.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/**
+ * @internal
+ */
+@Pipe({
+  standalone: true,
+  name: 'skyInputBoxHintText',
+})
+export class SkyInputBoxHintTextPipe implements PipeTransform {
+  public transform(
+    hintText?: string,
+    hostHintText?: string,
+  ): string | undefined {
+    if (hintText && hostHintText) {
+      return [hostHintText, hintText].join(' ');
```

Comment:
You could simplify with string concatenation, plus it's more performant than creating an array to perform the join.
```
`${hostHintText} ${hintText}`
```